src/zarr/core/array.py

```diff
@@ -1184,15 +1185,44 @@ def basename(self) -> str:
 
     @property
     def cdata_shape(self) -> ChunkCoords:
+        """
+        The shape of the chunk grid for this array. This property exists for backwards compatibility.
+        See :func:`chunk_grid_shape` for the preferred method.
+
+        Returns
+        -------
+        tuple[int, ...]
+            The shape of the chunk grid for this array.
+        """
+        return self.chunk_grid_shape
+
+    @property
+    def chunk_grid_shape(self) -> ChunkCoords:
```

Comment:
I added a new property called `chunk_grid_shape` because `cdata_shape` is ambiguous. `cdata_shape` is still around, but it now uses `chunk_grid_shape`